<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SailfishWizards::Internal::QmlOptionPage</class>
 <widget class="QWizardPage" name="SailfishWizards::Internal::QmlOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>QML File Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="baseTypeLabel">
       <property name="text">
        <string>Base type of QML document:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="baseTypeComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="VersionLabel">
       <property name="text">
        <string>Qt Quick version:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="VersionComboBox"/>
     </item>
     <item row="5" column="1">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
